zenoh-flow/src/types/message.rs

```diff
@@ -261,7 +261,7 @@ pub enum ControlMessage {
 
 /// The Zenoh-Flow message that is sent across `Link` and across Zenoh.
 ///
-/// It contains either a [`DataMessage`](`DataMessage`) or a [`Watermark`](`Watermark`).
+/// It contains either a [`DataMessage`](`DataMessage`) or a [`Watermark`](`uhlc::Timestamp`).
```

Comment:
I'd change the last sentence for:

It either contains a [`DataMessage`](`DataMessage`) or a [`Timestamp`](`uhlc::Timestamp`), signaling in the latter case that the message is a _Watermark_.

Reply:
👍